### ISSUE TITLE
change in ordering api

### ISSUE THREAD

**Issue by MiniXC** [MiniXC/simple-back#5]
At the moment the ordering is a bit clumsy and counter-intuitive (see example below).
````python
b.order_pct('MSFT', 0.5) # long 50% of value in MSFT (will fail if >50% are invested)
b.order_abs('MSFT', 1_000) # long 1_000$ MSFT
b.order_abs('MSFT', b.balance.current*0.5) # long 50% of available funds in MSFT

b.order_pct('MSFT', -1) # short 100% MSFT
...
````
# Proposed
````python
b.long('MSFT', percent=.5)
b.long('MSFT', absolute=1_000)
b.long('MSFT', percent_available=.5)
b.long('MSFT', nshares=1)
# equivalent for short
````
Any other suggestions? What (if anything) should it default to when the keyword isn't set?

**Comment by MiniXC** [comment]
Changed `num_shares` to `nshares`

**Comment by MiniXC** [comment]
Implemented in 51812e5530ec5ced7e7837b02f247b86a1fbdc66
